content/en/docs/setup/additional-setup/external-controlplane/index.md

```diff
@@ -70,272 +76,443 @@ $ export REMOTE_CLUSTER_NAME=<your remote cluster name>
+                  - port: 15021
+                    targetPort: 15021
+                    name: status-port
+                  - port: 15012
+                    targetPort: 15012
+                    name: tls-xds
+                  - port: 15017
+                    targetPort: 15017
+                    name: tls-webhook
+    EOF
+    {{< /text >}}
+
+    Then, install the gateway in the `istio-system` namespace of the external cluster:
+
+    {{< text bash >}}
+    $ istioctl install -f controlplane-gateway.yaml --context="${CTX_EXTERNAL_CLUSTER}"
+    {{< /text >}}
+    
+1. Run the following command to confirm that the ingress gateway is up and running:
+    
+    {{< text bash >}}
+    $ kubectl get po -n istio-system --context="${CTX_EXTERNAL_CLUSTER}"
+    NAME                                   READY   STATUS    RESTARTS   AGE
+    istio-ingressgateway-9d4c7f5c7-7qpzz   1/1     Running   0          29s
+    istiod-68488cd797-mq8dn                1/1     Running   0          38s
+    {{< /text >}}
+
+    You will notice an istiod deployment is also created in the `istio-system` namespace. This is used to configure the ingress gateway
+    and is NOT the control plane used by remote clusters.
+
+    {{< tip >}}
+    This ingress gateway could be configured to host multiple external control planes, in different namespaces on the external cluster,
+    although in this example you will only deploy a single external istiod in the `external-istiod` namespace.
+    {{< /tip >}}
+
+1. Configure your environment to expose the Istio ingress gateway service using a public hostname with TLS. Set the `EXTERNAL_ISTIOD_ADDR` environment variable to the hostname and `SSL_SECRET_NAME` environment variable to the secret that holds the TLS certs:
+
+    {{< text syntax=bash snip_id=none >}}
+    $ export EXTERNAL_ISTIOD_ADDR=<your external istiod host>
+    $ export SSL_SECRET_NAME=<your external istiod secret>
+    {{< /text >}}
+
+#### Set up the control plane in the external cluster
+
+1. Create the `external-istiod` namespace, which will be used to host the external control plane:
+
+    {{< text bash >}}
+    $ kubectl create namespace external-istiod --context="${CTX_EXTERNAL_CLUSTER}"
+    {{< /text >}}
+
+1. The control plane in the external cluster needs access to the remote cluster to discover services, endpoints,
+    and pod attributes. Create a secret with credentials to access the remote cluster’s `kube-apiserver` and install
+    it in the external cluster:
+
+    {{< text bash >}}
+    $ kubectl create sa istiod-service-account -n external-istiod --context="${CTX_EXTERNAL_CLUSTER}"
+    $ istioctl x create-remote-secret \
+      --context="${CTX_REMOTE_CLUSTER}" \
+      --type=config \
+      --namespace=external-istiod | \
+      kubectl apply -f - --context="${CTX_EXTERNAL_CLUSTER}"
+    {{< /text >}}
+
+1. Before you deploy the external istiod, you need to create the root namespace on the config (remote) cluster.
```

Comment:
Would it be possible for `istioctl install` to do this automatically, just like it does for local istio-system ns, so we can eliminate this step?